packages/core/src/utils/ans/utils/ans-name-to-lp-token-name.ts

```diff
@@ -0,0 +1,8 @@
+import { joinAssetNames } from './join-asset-names'
+
+export function ansNameToLpTokenName(
```

Comment:
```suggestion
export function lpTokenName(
```

Reply:
This would be more appropriately named `lpTokenName` based on `stakingContract`.

Reply:
Renamed to `lpTokenToAnsId`